### PR TITLE
Install Tekton Logs Collector using Kustomize generator

### PR DESCRIPTION
**Pull request by khrm**
ArgoCD CR's Application/ApplicationSet CR isn't available in every cluster. So we switched to using kustomize generator for vector installation.

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/assign @enarha

File: components/pipeline-service/production/base/main-pipeline-service-configuration.yaml [hunk 1]
```diff
@@ -601,7 +608,7 @@ data:
     LOGS_API=false
     LOGS_TYPE=File
     LOGS_BUFFER_SIZE=5242880
-    LOGS_PATH=//logs
+    LOGS_PATH=/logs
```
Comment (enarha) [inline components/pipeline-service/production/base/main-pipeline-service-configuration.yaml:611]: Why was that value changed? I see further below we also downgrade (reference older image) of Results, why?

**Review by khrm — COMMENTED** [review]
@enarha I have removed the prod changes.

**Comment by khrm** [comment]
Vector got deployed to Dev env.
image

/hold cancel

**Pull request merged**